ci/scripts/driver.sh

```diff
@@ -48,17 +48,19 @@ esac
 ######################################################
 # setup runtime env for correct python install and git
 ######################################################
+set +x
 source "${HOMEgfs}/ush/module-setup.sh"
 module use "${HOMEgfs}/modulefiles"
 module load "module_gwsetup.${MACHINE_ID}"
+set -x
 
 ############################################################
 # query repo and get list of open PRs with tags {machine}-CI
 ############################################################
 pr_list_file="open_pr_list"
-rm -f "${pr_list_file}"
+touch "${GFS_CI_ROOT}/${pr_list_file}"
```

Comment:
If this isn't being removed and two lines down you are just appending, how will old PRs ever be removed?